src/map_plot.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from folium.features import DivIcon
+import folium
+import json
+import sys
+
+# Given a VROOM solution with latlng locations, generates a map of the paths of the vehicles (up to 15 vehicles)
```

Comment:
I guess the 15 limit is related to the size of the `colors` list. It would be possible to loop through that list, pretty much as we do it in the `plot.py` script [here](https://github.com/VROOM-Project/vroom-scripts/blob/a6459e3de24b1a0e37afb79aeb6dd8ee278813b3/src/plot.py#L37).